Database/databaseHelpers/seedDB.js

```diff
@@ -104,7 +108,16 @@ function seedDatabase() {
         reviewObject.reviewTitle = lorem.generateWords(numberOfWords);
         console.log('reviewObject', reviewObject);
         db.create(reviewObject, function (err, small) {
-          if (err) return handleError(err);
+          attempted++;
+          if (err) {
+            console.error(err);
+          } else {
+            inserted++;
+          }
+          if (attempted >= total) {
+            console.log(`Database Seeded: attempted ${attempted} / ${total} - ${inserted} inserted`);
```

Comment:
Cool way of checking everything got seeded! You might be able to also just keep track of how many errors you get during creation / console log error at i too.